### DISCUSSION TITLE
Using WSLg as Standalone / without WSL?

### DISCUSSION THREAD

**Discussion by Roemer** [microsoft/wslg#1133] [General]
I am often working on Remote VMs with Linux and use VcXsrv as X-Server on Windows. As WSLg seems a bit more up-to-date and also directly works with WSL of course, I wonder if there is a possibility to just install WSLg without WSL and use that as X-Server?

**Comment by bavalpey** [comment]
Would love to know this as well. The existing X servers have pitiful documentation and poor support.

**Comment by ericblade** [comment]
I'd recommend MobaXTerm, I used that for a lot of years.

I'd guess you could just install a slim WSL installation with like minimal Ubuntu and X support, then use it to remote to your VMs, and use that to get WSLg involved.    

Otherwise, it'd probably be a difficult operation to get working -- WSLg uses Wayland, not X.

**Comment by bavalpey** [comment]
I kind of want to use vscode on Windows natively because it is seamless.
MobaXterm is fully fledged, right? There's no way I can hook into just the
X server it provides?

All I'm trying to do is get vscode's git feature to work with remote
development when I have GPG signing on.
Not really going to give up vscode for that.
I just made the move away from Neovim to vscode since terminal development
has basically stagnated.
